addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -567,34 +567,36 @@ def test_in_invoice_line_onchange_partner_1(self):
             {
                 **self.product_line_vals_1,
                 'partner_id': self.partner_b.id,
+                'account_id': self.product_b.property_account_expense_id.id,
+                'tax_ids': self.tax_purchase_b.ids,
             },
             {
                 **self.product_line_vals_2,
                 'partner_id': self.partner_b.id,
+                'tax_ids': self.tax_purchase_b.ids,
+                'price_total': 184.0,
             },
             {
                 **self.tax_line_vals_1,
                 'partner_id': self.partner_b.id,
-            },
-            {
-                **self.tax_line_vals_2,
-                'partner_id': self.partner_b.id,
+                'name': self.tax_purchase_b.name,
+                'tax_line_id': self.tax_purchase_b.id,
             },
             {
                 **self.term_line_vals_1,
-                'name': 'turlututu installment #1',
+                'name': 'turlututu',
                 'partner_id': self.partner_b.id,
                 'account_id': self.partner_b.property_account_payable_id.id,
-                'amount_currency': -338.4,
-                'credit': 338.4,
+                'amount_currency': -331.2,
+                'credit': 331.2,
```

Comment:
Did you check these changes, can you explain them?

Reply:
Hello @william-andre 
Sorry for late reply

I updated the test cases to reflect changes in the flow. 
Now, when there's a change in Fiscal position, we're also updating the associated taxes and accounts in draft journal entries. Previously, these changes weren't reflected in draft entries, but now they are.

I've modified four test cases since the journal entries for these tests were in draft state.

- Product A has one tax, `tax_sale_a`. 
- Product B has two taxes, `tax_sale_a` and `tax_sale_b`
- Here `tax_sale_b` is a copy of `tax_sale_a`. 
- The `fiscal_pos_a` changes `tax_sale_a` to `tax_sale_b`.

Before the fiscal change, product B's line had two taxes: `tax_sale_a` and `tax_sale_b`. This resulted in five journal items (two for the product, two for taxes, and one for Accounts Payable).

After the fiscal change, product B's line has only `tax_sale_b`. This reduces the number of journal items to four (two for the product, one for taxes, and one for Accounts Payable).

As a result, the journal items and the total amount have changed accordingly.

Reply:
Did you forget to push?